invenio_checks/services/schema.py

```diff
@@ -25,14 +28,31 @@ class Meta:
         unknown = EXCLUDE
 
     id = fields.UUID(dump_only=True)
-
-    created = TZDateTime(timezone=timezone.utc, format="iso", dump_only=True)
-    updated = TZDateTime(timezone=timezone.utc, format="iso", dump_only=True)
-
-    title = SanitizedUnicode(required=True)
-    description = SanitizedUnicode()
-
-    active = fields.Boolean(load_default=True)
+    community_id = fields.Str(required=True)
+    check_id = fields.Str(required=True)
+    params = fields.Dict(required=False, default={})
+    severity = fields.Str(required=False, default=Severity.INFO.value)
+    enabled = fields.Bool(required=False, default=True)
+
+    @validates("community_id")
+    def validate_community_id(self, value):
+        """Validate that the community_id is a valid UUID."""
+        try:
+            UUID(value)
```

Comment:
minor: it might be nice to check if the community exist. It could also be that it is the caller checking that.
`CommunityMetadata.query(CommunityMetadata.id == value).exists()`